src/components/CharacterBox/CharacterBox.js

```diff
@@ -56,10 +56,12 @@ export default class CharacterBox extends Component {
 
   generateCharacters = () => {
     return this.state.characters.map((character, ind) => {
+      const isFavorite = !!this.props.favorites.find(fav => fav.name === character.name)
```

Comment:
Just looking into the double !! - so you're wanting isFavorite to be a boolean, not a return of the character you are looking for?  You are just checking to see if it found something?

Reply:
Yes, I don't need object that was found I just need result (find or not) of this action.